javascript/ql/test/library-tests/NPM/src/node_modules/parent-modue/main.js

```diff
@@ -0,0 +1 @@
+export default "parent";
```

Comment:
You have a typo in the folder name.  
You have named the folder `parent-modue` instead of `parent-module`.  

I tried to run your test-case locally using `node` (to confirm that the behavior is correct), and it failed due to the typo.

Reply:
Oops, sorry.
I'll fix it soon.